### ISSUE TITLE
Code-In Logo not visible on smaller screens

### ISSUE THREAD

**Issue by shiftsayan** [fossasia/gci15.fossasia.org#443]
![screen shot 2016-01-15 at 1 09 07 pm](https://cloud.githubusercontent.com/assets/8451336/12347990/3e6afe72-bb89-11e5-822b-25320e3c3c30.png)
![screen shot 2016-01-15 at 1 09 09 pm](https://cloud.githubusercontent.com/assets/8451336/12347989/3e69d07e-bb89-11e5-963d-011f30a2e08d.png)

Due to the changes in the title bar, the GCI Logo does not appear on smaller screens. However, on making the text smaller (zooming out), it became visible.

**Comment by abishekvashok** [comment]
Can I work on this please have an idea in mind but in another task please wait till my task is approved then I will commit here! @shiftsayan or can I ?

**Comment by shiftsayan** [comment]
Yes, go ahead @Abhi2424shek

**Comment by abishekvashok** [comment]
If you are a mentor then please accept my task after reviewing I will give link and merge my pull request (only after reviewing) so that I would submit it now itself got the code ready! :) :+1:

**Comment by shiftsayan** [comment]
No, I am not a mentor :P

**Comment by Yathannsh** [comment]
@shiftsayan the Code-in logo has been deliberately removed from smaller screens as it overlaps with the Navigation on devices less than a width of 768px

**Comment by abishekvashok** [comment]
Ok @Yathannsh close this issue man!

**Comment by shiftsayan** [comment]
Alrighty @Yathannsh
